tests/unit/unit-util.cpp

```diff
@@ -229,6 +229,53 @@ TEST_P(UnitUtilDimacsTests, cnf_to_dimacs)
 }
 
 
+
+TEST_P(UnitUtilDimacsTests, tseitin)
+{
+  s->set_opt("incremental", "true");
+  Term p = s->make_symbol("p", boolsort);
+  Term q = s->make_symbol("q", boolsort);
+  Term r = s->make_symbol("r", boolsort);
+  Term t = s->make_symbol("t", boolsort);
+  
+  //a=((p or q) and r) implies (not t)
+  Term a = s->make_term(Implies, s->make_term(And, s->make_term(Or, p, q), r), s->make_term(Not, t));
+  Term cnf1 = to_cnf(a, s);
+  string st1 = cnf1->to_string();
+  string ans1 =
```

Comment:
It would be much better to compare terms and not strings, in case we change something in the string representation at some point. Could you construct a `Term` object that represents the expected result and compare it to the actual result (and not to its string representation)?

Reply:
I will not be able to create the same symbols using the same solver(As an exception would be thrown). Should I use 2 solvers?

Reply:
Yes, good point!

Reply:
But actually, I think that the terms will not be equal if they are created by two different solvers, but they should be based on the same solver, that is, two instances of cvc4, or two instances of z3, etc.

Reply:
Ok!

Reply:
Should I create 2 solvers s1, s2(of the same solver) in the class of the test, or is that not allowed?